package.json

```diff
@@ -26,7 +26,7 @@
     "mkdirp": "^0.5.1",
     "recursive-readdir": "^2.2.2",
     "targz": "^1.0.1",
-    "web3": "^1.0.0-beta.34",
+    "web3": "1.0.0-beta.34",
```

Comment:
Any reason for pinning this version?

Reply:
Yes; `>= web3@1.0.0-beta.39` fails on fetching repo info (`apm.js` needs to be upgraded).